TrenDAP/wwwroot/TypeScript/global.d.ts

```diff
@@ -94,7 +94,7 @@ export namespace DataSourceTypes {
         DataSourceTypeID: number,
         URL: string,
         RegistrationKey: string,
-        Expires: string,
+        Expires: string | null,
```

Comment:
```suggestion
        Expires?: string,
```

Reply:
We changed C# Model to `Enabled?`

Reply:
The expires field no longer exists up the waterfall.